catalog/ui/search-ui/search-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -23,6 +23,9 @@
         <AD description="Disables relevance and distance normalization." name="Disable Normalization"
             id="normalizationDisabled" required="false" type="Boolean" default="false"/>
 
+        <AD description="Controls the maximum number of results returned from a query." name="Maximum Query Results"
```

Comment:
✏️  Maybe include what maximum results means. Like, what happens when I have more results than this maximum? Why set a maximum in the first place?